CONTRIBUTING.md

```diff
@@ -22,18 +22,17 @@ be added in modules. Exceptions would be things like new cross-OS providers
 and updates to existing core types.
 
 If you are unsure of whether your contribution should be implemented as a
-module or part of Puppet Core, you may visit
-[#puppet-dev on Freenode IRC](https://freenode.net) or ask on the
-[puppet-dev mailing list](https://groups.google.com/forum/#!forum/puppet-dev)
-for advice.
+module or part of Puppet Core, you may visit [#puppet-dev on Freenode IRC](https://freenode.net),
```

Comment:
We can probably drop IRC from this list. In talking with Kara, we probably want to have no more than two of IRC/Slack/mailing list/etc. and Slack & the mailing list are the ones that are being promoted/focused on now.